src/PresentationalComponents/SimpleTableFilter/SimpleTableFilter.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import { Button, ButtonVariant } from '@patternfly/react-core';
+
+class SimpleFilter extends Component {
+  constructor(props) {
+    super(props);
+    this.onInputChange = this.onInputChange.bind(this);
+    this.onFilterSubmit = this.onFilterSubmit.bind(this);
+    this.state = {
+      activeFilter: ''
+    }
+  }
+
+  onInputChange(event) {
+    this.setState({ activeFilter: event.target.value });
+    this.props.hasOwnProperty('onFilterChange') && this.props.onFilterChange(this.state.activeFilter);
+  }
+
+  onFilterSubmit() {
+    this.props.hasOwnProperty('onButtonClick') && this.props.onButtonClick(this.state.activeFilter);
+  }
+
+  render() {
+    const {
+      placeholder = 'Search items',
+      buttonTitle = 'Filter',
+      className = '',
+      onButtonClick,
+      onFilterChange,
+      ...props
+    } = this.props;
+    return (
+      <div className={`pf-c-input-group ${className}`} {...props}>
+        <input className="pf-c-form-control" type="text" placeholder={placeholder} onChange={this.onInputChange}/>
+        {buttonTitle && <Button variant={ButtonVariant.secondary} onClick={this.onFilterSubmit}>{buttonTitle}</Button>}
+      </div>
+    )
+  }
+}
+
+SimpleFilter.propTypes = {
+  buttonTitle: PropTypes.string,
+  placeholder: PropTypes.string,
+  className: PropTypes.string,
+  onButtonClick: PropTypes.func,
+  onFilterChange: PropTypes.func
+}
```

Comment:
Do we want to require any of these? Couldn't someone just clear out the placeholder string or buttonTitle and have blank spaces?

Reply:
These proptypes are for users of this component so they know what props and what values to pass into. Of course if somebody doesn't want to have placeholder he can pass empty string, if somebody doesn't want to have button they can just pass ampty string into `buttonTitle` and no button will be visible.